server/src/test/java/com/broadcom/lsp/cobol/core/engine/CobolLanguageEngineTest.java

```diff
@@ -132,18 +132,21 @@ void testLanguageEngineRun() {
                         List.of()))));
 
     ResultWithErrors<SemanticContext> expected =
-        new ResultWithErrors<>(
-            new SemanticContext(
-                Map.of(),
-                Map.of(),
-                Map.of(),
-                Map.of(),
-                getConstantDefinitions(),
-                Map.of(),
-                Map.of(),
-                Map.of(),
-                expectedOutlineTree),
-            List.of(error));
+            new ResultWithErrors<>(
+                    SemanticContext.builder()
+                            .constantDefinitions(getConstantDefinitions())
+                            .outlineTree(expectedOutlineTree)
```

Comment:
Defining Builder with default values(Map.of()) would help here not to repeat Map.of()